2020-12-21 17:08:12,762 — __main__ — INFO — <module>:11 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:08:13,023 — __main__ — INFO — <module>:17 — Done
2020-12-21 17:30:13,343 — __main__ — INFO — <module>:11 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:30:13,344 — __main__ — INFO — <module>:17 — Done
2020-12-21 17:37:40,143 — __main__ — INFO — <module>:11 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:37:40,144 — __main__ — INFO — <module>:17 — Done
2020-12-21 17:41:17,262 — __main__ — INFO — <module>:11 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:41:17,264 — __main__ — INFO — <module>:17 — Done
2020-12-21 17:44:42,923 — __main__ — INFO — <module>:11 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:44:42,924 — __main__ — INFO — <module>:17 — Done
2020-12-21 17:48:36,307 — __main__ — INFO — <module>:11 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:48:36,308 — __main__ — INFO — <module>:17 — Done
2020-12-21 17:49:08,342 — __main__ — INFO — <module>:11 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:49:08,343 — __main__ — INFO — <module>:17 — Done
2020-12-21 17:54:32,816 — __main__ — INFO — <module>:12 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:54:32,818 — api.data_getter — INFO — get_data_async:38 — Getting data.
2020-12-21 17:54:32,818 — api.data_getter — ERROR — get_data_async:64 — Ops Something Else: 'CptecApiSuite' object has no attribute 'items'
2020-12-21 17:54:32,819 — __main__ — INFO — <module>:18 — Done
2020-12-21 17:56:33,735 — __main__ — INFO — <module>:12 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:56:51,978 — api.data_getter — INFO — get_data_async:38 — Getting data.
2020-12-21 17:56:51,979 — api.data_getter — ERROR — get_data_async:64 — Ops Something Else: 'CptecApiSuite' object has no attribute 'items'
2020-12-21 17:56:52,593 — __main__ — INFO — <module>:18 — Done
2020-12-21 17:57:03,853 — __main__ — INFO — <module>:12 — {'current_weather_cond_capitals': 'http://servicos.cptec.inpe.br/XML/capitais/condicoesAtuais.xml'}
2020-12-21 17:59:08,953 — api.data_getter — INFO — get_data_async:38 — Getting data.
2020-12-21 17:59:09,372 — api.data_getter — ERROR — get_data_async:64 — Ops Something Else: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 17:59:17,480 — api.data_getter — INFO — get_data_async:38 — Getting data.
2020-12-21 17:59:17,863 — api.data_getter — ERROR — get_data_async:64 — Ops Something Else: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 18:12:03,930 — api.data_getter — ERROR — get_data_async:64 — Ops Something Else: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 18:12:03,934 — __main__ — INFO — <module>:18 — Done
2020-12-21 18:14:16,927 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 18:15:05,805 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: unhashable type: 'dict'
2020-12-21 18:15:42,819 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 18:16:59,013 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: append() takes exactly one argument (2 given)
2020-12-21 18:17:13,071 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: unhashable type: 'list'
2020-12-21 18:24:01,520 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: unhashable type: 'list'
2020-12-21 18:25:13,778 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: unhashable type: 'list'
2020-12-21 18:25:37,298 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: unhashable type: 'list'
2020-12-21 18:25:59,398 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: unhashable type: 'list'
2020-12-21 18:26:51,581 — api.data_getter — ERROR — get_data_async:69 — Ops Something Else: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 18:42:56,058 — __main__ — INFO — <module>:18 — Done
2020-12-21 19:25:55,064 — __main__ — INFO — <module>:18 — Done
2020-12-21 19:26:05,760 — __main__ — INFO — <module>:18 — Done
2020-12-21 19:27:18,945 — __main__ — INFO — <module>:20 — Done
2020-12-21 19:27:44,797 — __main__ — INFO — <module>:20 — Done
2020-12-21 19:29:55,944 — api.data_getter — ERROR — get_data:84 — Error: 'bytes' object is not callable
2020-12-21 19:34:29,689 — api.data_getter — ERROR — get_data:84 — Error: 'bytes' object is not callable
2020-12-21 19:39:42,554 — api.data_getter — ERROR — get_data:84 — Error: 'bytes' object is not callable
2020-12-21 19:40:36,837 — __main__ — INFO — <module>:20 — Done
2020-12-21 19:40:48,258 — __main__ — INFO — <module>:20 — Done
2020-12-21 19:51:34,604 — __main__ — INFO — <module>:20 — Done
2020-12-21 19:54:36,466 — __main__ — INFO — <module>:20 — Done
2020-12-21 19:58:38,838 — __main__ — INFO — <module>:20 — Done
2020-12-21 19:59:24,810 — __main__ — INFO — <module>:20 — Done
2020-12-21 20:01:44,367 — api.data_getter — ERROR — async_request:64 — Ops Something Else: object bytes can't be used in 'await' expression
2020-12-21 20:01:44,368 — api.data_getter — ERROR — get_data:84 — Error: object bytes can't be used in 'await' expression
2020-12-21 20:03:47,233 — api.data_getter — ERROR — async_request:64 — Ops Something Else: object bytes can't be used in 'await' expression
2020-12-21 20:03:47,234 — api.data_getter — ERROR — get_data:84 — Error: object bytes can't be used in 'await' expression
2020-12-21 20:04:17,938 — api.data_getter — ERROR — get_data:84 — Error: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 20:10:12,350 — __main__ — INFO — <module>:20 — Done
2020-12-21 20:13:32,154 — api.data_getter — ERROR — get_data:84 — Error: 'coroutine' object has no attribute 'content'
2020-12-21 20:23:57,549 — api.data_getter — ERROR — get_data:84 — Error: An asyncio.Future, a coroutine or an awaitable is required
2020-12-21 20:25:01,649 — api.data_getter — ERROR — async_request:64 — Ops Something Else: 'coroutine' object has no attribute 'content'
2020-12-21 20:25:01,650 — api.data_getter — ERROR — get_data:84 — Error: 'coroutine' object has no attribute 'content'
2020-12-21 20:25:27,173 — api.data_getter — ERROR — async_request:64 — Ops Something Else: 'coroutine' object has no attribute 'content'
2020-12-21 20:25:27,173 — api.data_getter — ERROR — get_data:84 — Error: 'coroutine' object has no attribute 'content'
